packages/angular_devkit/build_angular/src/angular-cli-files/plugins/index-html-webpack-plugin.ts

```diff
@@ -67,6 +65,7 @@ export class IndexHtmlWebpackPlugin {
       (compilation as compilation.Compilation & { fileDependencies: Set<string> })
```

Comment:
I think the sorting logic needs to move downwards, in the index generator, as otherwise, we'll have two place that have sorting logic.

As mentioned, the sorting logic, should not happened at `@angular/build-webpack`, as that can be used with any webpack config and if not aware of any logic with `build-angular`, and not aware that `polyfills` entrypoint should take priority over `main`, this logic should really happen when generating the index, which is the sole place that will be used by all builders to generate an index.

Reply:
Good point. That means we move it to the `generate-index-html.ts`, right?

Reply:
yes